python/cudf/setup.py

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) 2018-2021, NVIDIA CORPORATION.
-
+# modify on purpose :P
```

Comment:
But eventually unmodify on purpose please :)

Reply:
From a quick look through the logic in the copyright I'm not sure that it will flag this as an error. I think you have to either 1) have an invalid year, 2) have a later year before an earlier year (e.g. 2020-2018), or 3) if there is a line before the copyright line. I don't think it is a general check for the entire copyright.

Reply:
Removed this on purpose :P